gpcontrib/arenadata_toolkit/src/arenadata_toolkit_guc.c

```diff
@@ -0,0 +1,295 @@
+#include "arenadata_toolkit_guc.h"
+
+
+#include "cdb/cdbvars.h"
+#include "catalog/objectaccess.h"
+#include "catalog/pg_db_role_setting.h"
+#include <limits.h>
+#include "utils/guc.h"
+#include "tf_shmem.h"
+
+#define DEFAULT_BLOOM_SIZE 1000000
```

Comment:
For defines of sizes, periods I suggest to add units in the name, for ex.:
`DEFAULT_BLOOM_SIZE` -> `DEFAULT_BLOOM_SIZE_BYTES`